src/posts/2022-03-05-troubleshoot-cilium-network-policy/index.md

```diff
@@ -0,0 +1,13 @@
+---
+date: '2022-03-05T17:00:00.000Z'
+title: 'Kubernetes Security — Control pod to pod communications with Cilium network policies'
+ogImageUrl: 'https://github.com/cilium/cilium.io/blob/add-network-policy-blog-post/src/posts/2022-03-05-troubleshoot-cilium-network-policy/brokenhubble.png'
```

Comment:
Please don't load the image from Github. Load it from the server.

Reply:
what do you mean "load it from the server"? Do you mean from medium? The problem with that is it has a * in the URL so it breaks markdown https://miro.medium.com/max/1400/1*WaXOtrm9c-HO1tH-_K9VUg.png

Reply:
No, I mean since you upload it to the cilium.io server anyway, you can also embed it directly, with a relative link. But I saw that you now reference the medium article which is fine I think.